src/views/List.jsx

```diff
@@ -1,16 +1,48 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+
+	const handleChange = (e) => {
+		setSearchItem(e.target.value);
+	};
+
+	const filteredItems = data.filter((item) => {
+		return item.name.toLowerCase().includes(searchItem.toLowerCase());
+	});
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<label htmlFor="item-search"> Search for an item: </label>
+			<input
+				id="item-search"
+				type="text"
+				placeholder="Search item..."
+				onChange={handleChange}
+				value={searchItem}
+			/>
+			{searchItem && (
+				<button
+					type="button"
+					name="clearInput"
+					aria-label="Clear input"
```

Comment:
This is great here since the button is just an `x`. If our button said `Clear Input`, the aria-label wouldn't be needed. But that's just for informational purposes. Keep it as-is 😄